src/Microsoft.FeatureManagement/ConfigurationFields.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+namespace Microsoft.FeatureManagement
+{
+    internal static class ConfigurationFields
+    {
+        // Enum keywords
+        public const string RequirementType = "RequirementType";
+
+        // Feature filters keywords
+        public const string FeatureFiltersSectionName = "EnabledFor";
+        public const string FeatureFilterConfigurationParameters = "Parameters";
+
+        // Other keywords
+        public const string NameKeyword = "Name";
+        public const string FeatureManagementSectionName = "FeatureManagement";
+        public const string FeatureFlagsSectionName = "FeatureFlags";
+
+        // Azure App Configuration feature flag schema keywords
+        public const string FeatureFlagId = "id";
+        public const string FeatureFlagEnabled = "enabled";
+        public const string FeatureFlagConditions = "conditions";
+        public const string FeatureFlagClientFilters = "client_filters";
+        public const string LowercaseNameKeyword = "name";
```

Comment:
Why do we need lower case name? .NET configuration system is case insensitive.

Reply:
（°o°；）Really? I never noticed that. Thanks for pointing it out!

Reply:
Got pulled away before I could write another comment. It would be better to have App Configuration fields in a separate file.